prwkv/modelrun.py

```diff
@@ -1,234 +1,251 @@
-########################################################################################################
-# The RWKV Language Model - https://github.com/BlinkDL/RWKV-LM
-########################################################################################################
 
-import types
 import torch
-import math, os, gc
```

Comment:
instead of changing this file, we can create a new file modelrunoptimized.py and modelrun.py (research variant) that way people have a choice for the backend